### PR TITLE
Changes to the Blacklist

### PR DESCRIPTION
**Pull request by tobiasreichel**
Change of BLACKLISTED_CFTOOL_IDS to BLACKLISTED_STEAM64_IDS

cause its the using steam id to filter and not cftools id

### CODE REVIEW

**Comment by Ariastarcos** [comment]
Thanks for raising the issue and contributing to the repository.

The reason I went with BLACKLISTED_CFTOOL_IDS for the naming convention is that I initially planned to make both Steam64 and CFTools id's blacklistable. And we obviously can't go naming stuff BLACKLISTED_CFTOOL_OR_STEAM64_IDS. I'll rename the variable to BLACKLISTED_IDS and a comment in the README explaining the options to avoid confusion - I'll also go and implement support for both blacklisting steam64's and CFTools id's.

File: src/pages/index.jsx [hunk 1]
```diff
@@ -365,12 +365,12 @@ export async function getServerSideProps ({ query }) {
...
-    config.BLACKLISTED_CFTOOLS_IDS
-    && Array.isArray(config.BLACKLISTED_CFTOOLS_IDS)
-    && config.BLACKLISTED_CFTOOLS_IDS[0]
+    config.BLACKLISTED_STEAM64_IDS
+    && Array.isArray(config.BLACKLISTED_STEAM64_IDS)
+    && config.BLACKLISTED_STEAM64_IDS[0]
   ) {
     // Filter out blacklisted id's
-    const filteredRes = res.filter(({ id }) => !config.BLACKLISTED_CFTOOLS_IDS.includes(id));
+    const filteredRes = res.filter(({ id }) => !config.BLACKLISTED_STEAM64_IDS.includes(id));
```
Comment (Ariastarcos) [inline src/pages/index.jsx]: The `id` property is destructured from the CFTools API, the id represents a CFTools API id, **not** a steam64 - therefor, I have marked this pull request incorrect

File: src/pages/index.jsx [hunk 2]
```diff
@@ -385,7 +385,7 @@ export async function getServerSideProps ({ query }) {
     if (
       stats
       && config.ALLOW_PLAYER_STATISTICS_FOR_BLACKLIST === false
-      && config.BLACKLISTED_CFTOOLS_IDS.includes(query.steam64)
+      && config.BLACKLISTED_STEAM64_IDS.includes(query.steam64)
```
Comment (Ariastarcos) [inline src/pages/index.jsx]: I'm assuming this is what made you raise the issue. But the truth is that, even though the query parameter is defined as `steam64`, we automatically resolve a bunch of id's behind the scenes by using the cftools-sdk `SteamId64` and `CFToolsId` constructors. We could possibly even support Bohemia Interactive and BattlEye ids - but I'm not sure if those are even used by normal players

I'll fix the naming convention - sorry about the confusion

File: src/pages/index.jsx (hunk 1 above)
Comment (Ariastarcos) [reply src/pages/index.jsx]: Here is proof the id property represents a CFTools id as this is clearly not a steam64 being all numbers.

Code_jKLRnOhV3d

File: src/pages/index.jsx (hunk 2 above)
Comment (tobiasreichel) [reply src/pages/index.jsx]: okay good to know with the change cause it only took the steamid and not cf id

Comment (tobiasreichel) [reply src/pages/index.jsx]: and bi id and be id is not really used in the community

**Comment by Ariastarcos** [comment]
I still agree the naming convention is confusing and I'm resolving that in this PR

**Pull request merged**